nusight2/src/client/components/localisation/robot_model.ts

```diff
@@ -158,7 +162,8 @@ export class LocalisationRobotModel {
       Hfw: Matrix4.of(),
       Rwt: Quaternion.of(),
       motors: ServoMotorSet.of(),
-      fieldLinesDots: { rPWw: [] },
+      fieldLinePoints: { rPWw: [] },
+      ball: { rBWw: Vector3.of() },
```

Comment:
This means all robots will draw a ball at the center of the field, even if it hasn't seen a ball.

Lets make it optional on the model instead. i.e.

```ts
@observable ball?: { rBWw: Vector3 };
```